the bug #60 not solved in rc.2

Hello,

Thanks for help to solve the #60 but again in the rc.2 same error....
```
> express-typescript-boilerplate@3.0.0-rc.2 setup /Users/far-paul/Sites/lucy/outilSL/server2
> npm start setup.db


> express-typescript-boilerplate@3.0.0-rc.2 start /Users/far-paul/Sites/lucy/outilSL/server2
> nps "setup.db"

Scripts must resolve to strings. There is no script that can be resolved from "setup.db" https://github.com/kentcdodds/nps/blob/v5.7.1/other/ERRORS_AND_WARNINGS.md#missing-script
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! express-typescript-boilerplate@3.0.0-rc.2 start: `nps "setup.db"`
```
Thanks

--- a/package.json
+++ b/package.json
@@ -8,7 +8,7 @@
     "test": "npm start test",
     "build": "npm start build",
     "presetup": "yarn install",
-    "setup": "npm start setup.db"
+    "setup": "npm start config && npm start setup.script"
   },
   "engines": {
     "node": ">=8.0.0"

